source/server/session_repository.h

```diff
@@ -24,19 +24,18 @@ class SessionRepository {
   ~SessionRepository();
 
   typedef std::function<int32_t()> InitFunc;
-  typedef std::function<void(uint32_t)> CleanupSessionFunc;
+  typedef std::function<void(const std::string&)> CleanupSessionFunc;
 
   int add_session(
-      const std::string& session_name,
+      std::string& session_name,
       InitFunc init_func,
       CleanupSessionFunc cleanup_func,
-      uint32_t& session_id,
       SessionInitializationBehavior initialization_behavior = SESSION_INITIALIZATION_BEHAVIOR_UNSPECIFIED,
       bool* initialized_new_session = nullptr);
-  uint32_t access_session(uint32_t session_id, const std::string& session_name);
-  void remove_session(uint32_t id);
+  std::string access_session(const std::string& session_name);
+  void remove_session(const std::string& name);
 
-  void register_dependent_session(uint32_t id, uint32_t dependent_session_id, std::function<void()> cleanup);
+  void register_dependent_session(const std::string& id, const std::string& dependent_session_id, std::function<void()> cleanup);
```

Comment:
Nit: Not sure if it functionally matters, but can you match the name updates to what's done in the `session_repository.cpp` file: `id` -> `name` and `dependent_session_id` -> `dependent_session_name`?

Reply:
Not sure how I missed these when I did my search. I will update.

Reply:
I removed the usages of id and session_id.